training-maquetado/index.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+<body>
+
+<div class="title-box">
```

Comment:
is this div necessary? maybe you can add all the styles of the box to the title itself